docs-core/src/main/resources/db/update/dbupdate-028-0.sql

```diff
@@ -0,0 +1,14 @@
+create memory table T_REVIEW (
+    REV_USERNAME_C varchar(36),
```

Comment:
Should we make the "REV_USERNAME_C", "REV_DOCID_C", "REV_REVIEWERID_C" to be foreign keys?

Reply:
Ehhh... I vote let's avoid the headache for the sake of our collective convenience.